app/controllers/bulk_edit_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class BulkEditController < ApplicationController
+  include Blacklight::SearchHelper
+
+  def resources_edit
+    (@solr_response, @document_list) = search_results(q: params["q"], f: params["f"])
+  end
+
+  def resources_update
+    (_, document_list) = search_results(q: params["search_params"]["q"], f: params["search_params"]["f"])
```

Comment:
Does this retrieve the whole set of results, or just the first page?

Reply:
good question; we will look into this.

Reply:
I'd recommend adding a FL key with "id" too if that works through BL.

Reply:
can't seem to provide `fl` to the search builder.